src/apps/xpubs/xpubs.py

```diff
@@ -307,6 +317,33 @@ async def save_menu(self, show_screen):
             return self.export_coldcard
         return None
 
+    async def export_multiple_accounts_xpubs(self, from_account, to_account, file_format, show_screen):
+        if from_account > to_account:
+            await show_screen(Alert("Invalid Input!", "From account number should be less than or equal to To account number", button_text="Close"))
+            return
+        if to_account > 0x80000000:
+            raise AppError('Account number too large')
+        filedata = ""
+        for account in range(from_account, to_account+1):
+            self.account = account
```

Comment:
Why do you update `self.account`  here?
I was surprised when after exporting 10 accounts my default account became 10. I'd expect that `self.account` doesn't change after using this feature.
Just use `account` in the loop instead of `self.account`.

Reply:
OK, I think I copied some existing code and did not understood this difference.  I will change.